generator/src/main/java/com/scottlogic/deg/generator/inputs/AtomicConstraintReaderLookup.java

```diff
@@ -190,8 +191,22 @@
                 });
     }
 
+    private static <T> T throwIfValueNull(T value) throws InvalidProfileException {
+        if (value == null) {
+            throw new InvalidProfileException("Couldn't recognise 'value' property, it must be set to a value");
+        }
+        return value;
+    }
+
+    private static <T> Collection<T> throwIfValuesNull(Collection<T> values) throws InvalidProfileException {
+        if (values.contains(null)) {
+            throw new InvalidProfileException("Couldn't recognise 'values' property, it must not contain 'null'");
```

Comment:
This should be checking if the collection is null, not whether it contains null, this is checked elsewhere.

Reply:
The message should be updated here